spec/presenters/contact_presenter_spec.rb

```diff
@@ -61,6 +61,20 @@
 
       links = ContactPresenter.new(contact).links
 
+      expect(links[:links]["related"]).to eq([content_id])
+      expect(links[:links]["parent"]).to eq([contact.organisation.content_id])
+    end
+  end
+
+  context "#links for HMRC contact" do
+    let(:contact) { create :contact, :for_hmrc }
+
+    it "returns links data with HMRC contact's finder" do
+      content_id = SecureRandom.uuid
```

Comment:
Minor: could we assign a `related_contact` here and use that later on?

Reply:
Good shout! Realised there was a trait that adds related contacts. Also removed a redundant test.